mapper_visitor/src/builder.rs

```diff
@@ -0,0 +1,56 @@
+use crate::*;
+use chrono::{DateTime, FixedOffset};
+
+pub struct MeasurementBuilder<'a, T: MeasurementVisitor> {
+    visitor: &'a mut T,
+}
```

Comment:
This struct seems useless. This builder simply forwards all the call to the underlying visitor.

* Why not a single interface?
* I.e. the visitor trait with all the methods renamed with the `visit_` prefix.

Reply:
I felt so as well. It adds just one more wrapper.